public_suffix_list.dat

```diff
@@ -11052,6 +11052,17 @@ webhop.org
 worse-than.tv
 writesthisblog.com
 
+// ddnss.de . : https://www.ddnss.de/
+// Submitted by Robert Niedziela <webmaster@ddnss.de>
+ddnss.de
+dyn.ddnss.de
+dyndns.ddnss.de
+dyndns1.de
+dyn-ip24.de
+home-webserver.de
+dyn.home-webserver.de
+myhome-server.de
+ddnss.org
```

Comment:
There should be a newline between the last domain here for yours and the start of the next section.